diagnostic.md

```diff
@@ -35,7 +39,10 @@ Record your responses inside the fenced code blocks below each question.
     ```
 
     ```md
-    <!-- your response here -->
+    One difference is in the first block, product is nested inside of products.
```

Comment:
Yes, nested route definitions, do require a nested file structure.  This means that the Route and Template files for `'product'` are expected to be found at `app/products/product`.